examples/status-message-collections/src/lib.rs

```diff
@@ -37,14 +37,14 @@ impl StatusMessage {
 mod tests {
     use super::*;
     use near_bindgen::MockedBlockchain;
-    use near_bindgen::{testing_env, Config, VMContext};
+    use near_bindgen::{testing_env, VMContext};
 
     fn get_context(input: Vec<u8>, is_view: bool) -> VMContext {
         VMContext {
-            current_account_id: "alice.near".to_string(),
```

Comment:
motivation? (just want to be sure I'm on track with)

Reply:
We prohibited `.` in account names long time ago. I simple have not updated this code yet.